### DISCUSSION TITLE
Controlling D1 dimmer buzzer

### DISCUSSION THREAD

**Discussion by 420pootang69** [arendst/Tasmota#11419] [Ideas]
Hi All,

I asked this in the issue thread relating to the D1 dimmer, but seeing as that is closed, I thought it made more sense to put it here where it might get some visibility.

I'm interested in making the buzzer in the SonOff D1 dimmer functional in Tasmota.

From poking around, it appears to be controlled from U3, which is the BB10.

This does this mean it needs a serial command to be activated?

If so, would I be better breaking a track and just connecting it to an unused ESP pin? (weary of current draw, I know I could shove a transistor in there somewhere as well).

Here's the link to the GitHub issue in case anyone isn't familiar:
https://github.com/arendst/Tasmota/issues/7598

**Comment by sfromis** [comment]
As explained on Discord, the base assumption is that there just is no such command, as it would be useless to the factory setup.
